timely/src/progress/change_batch.rs

```diff
@@ -251,7 +272,7 @@ impl<T:Ord> ChangeBatch<T> {
     }
 
     /// Expose the internal vector of updates.
-    pub fn unstable_internal_updates(&self) -> &Vec<(T, i64)> { &self.updates }
+    pub fn unstable_internal_updates(&self) -> &[(T, i64)] { &self.updates[..] }
```

Comment:
I'm not sure what this change was about, but I think this reduces functionality (we can no longer check the internal capacity). I bet I thought I was just tidying, but I think I'll skip doing this.